doc/user/content/sql/create-source/_index.md

```diff
@@ -12,6 +12,7 @@ menu:
 ---
 
 A [source](../../overview/key-concepts/#sources) describes an external system you want Materialize to read data from, and provides details about how to decode and interpret that data. To create a source, you must specify a [connector](#connectors), a [format](#formats) and an [envelope](#envelopes).
+Like other relations, sources are namespaced by a database and schema.
```

Comment:
nit: Might be nice to make `namespaced` a link to https://materialize.com/docs/sql/namespaces/.